src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SIGN = "\\W+";
+    private static final String SPACE = " ";
+    private static final String SEPARATOR = System.lineSeparator();
+
+    public static void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Please don't use static methods unless there is particular reason for this. Static method are generally harder to test and maintain
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
```